backend/kesaseteli/applications/models.py

```diff
@@ -473,6 +576,76 @@ def birthdate(self) -> date:
         """
         return social_security_number_birthdate(self.social_security_number)
 
+    @property
+    def is_9th_grader_age(self) -> bool:
+        """
+        If applicant's age correct for a ninth grader ("9. luokkalainen" in Finnish)?
+        """
+        return (self.created_at.year - self.birthdate.year) == 16  # e.g. 2022 - 2006
+
+    @property
+    def is_upper_secondary_education_1st_year_student_age(self) -> bool:
+        """
+        If applicant's age correct for an upper secondary education first year student
+        ("Toisen asteen ensimmäisen vuoden opiskelija" in Finnish)?
+        """
+        return (self.created_at.year - self.birthdate.year) == 17  # e.g. 2022 - 2005
+
+    @property
+    def is_social_security_number_valid_according_to_vtj(self) -> bool:
```

Comment:
Good job naming the properties descriptively!

Reply:
Thanks!